.github/workflows/auto-release.yaml

```diff
@@ -0,0 +1,58 @@
+on:
+  pull_request:
+name: auto-release
+jobs:
+  approve:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/github-script@v3.0.0
+      with:
+        github-token: ${{secrets.GITHUB_TOKEN}}
+        script: |
+          // only approve PRs from release-please
+          if (context.payload.sender.login != "release-please") {
```

Comment:
I think the user is `release-please[bot]`

Reply:
done